### PR TITLE
Show personalization containers when consent is out

### PR DESCRIPTION
**Pull request by jonsnyder**
## Description

- When there is a consent cookie set to "out", do not apply the prehiding style and remove the prehiding snippet.
- When there is no consent cookie, and setConsent is called with "out", remove the prehiding style and prehiding snippet.

## Related Issue

https://jira.corp.adobe.com/browse/PDCL-11517

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

File: src/core/consent/createConsentStateMachine.js [hunk 1]
```diff
@@ -111,6 +111,22 @@ export default ({ logger }) => {
     awaitConsent: awaitInitial,
     withConsent() {
       return this.awaitConsent(true);
+    },
+    current() {
+      switch (this.awaitConsent) {
+        case awaitInDefault:
+          return { state: "in", wasSet: false };
```
Comment (jfkhoury) [inline src/core/consent/createConsentStateMachine.js:118]: Is the goal to differentiate between default consent and set consent by the visitor?

**Review by jfkhoury — APPROVED** [review]
Looks good Jon. Do you think we need to update the docs?

**Review by dompuiu — APPROVED** [review]

File: src/core/consent/createConsentStateMachine.js (hunk 1 above)
Comment (jonsnyder) [reply src/core/consent/createConsentStateMachine.js:118]: Yes. We only want to show containers when we have a "out" cookie.

File: src/components/Personalization/createFetchDataHandler.js [hunk 2]
```diff
@@ -20,13 +20,17 @@ export default ({
...
-  notificationHandler
+  notificationHandler,
+  consent
 }) => {
   return ({ cacheUpdate, personalizationDetails, event, onResponse }) => {
-    if (personalizationDetails.isRenderDecisions()) {
-      hideContainers(prehidingStyle);
-    } else {
-      showContainers();
+    const { state, wasSet } = consent.current();
```
Comment (ninaceban) [inline src/components/Personalization/createFetchDataHandler.js:27]: do we reach here if consent isn't given or out?

Comment (jonsnyder) [reply src/components/Personalization/createFetchDataHandler.js:27]: Yes because the awaitConsent happens after this.

**Pull request merged**